Common/UnitDefinitions/Length.json

```diff
@@ -293,6 +293,59 @@
           "Abbreviations": [ "h", "hh" ]
         }
       ]
+    },
+    {
+      "SingularName": "AstronomicalUnit",
+      "PluralName": "AstronomicalUnits",
+      "BaseUnits": {
+        "L": "Meter"
+      },
+      "FromUnitToBaseFunc": "x * 1.4959787070e11",
+      "FromBaseToUnitFunc": "x / 1.4959787070e11",
+      "XmlDocSummary": "One Astronomical Unit is the distance from the solar system Star, the sun, to planet Earth.",
+      "XmlDocRemarks": "https://en.wikipedia.org/wiki/Astronomical_unit",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "au", "ua", "AU" ]
```

Comment:
For parsing, abbreviations support case-insensitive parsing (unless ambiguous with other units' abbreviations). The only reason to keep `"AU"` is if you want to do `ToString("a2")`, but I don't think this will be used by anyone and that it can be removed.

https://github.com/angularsen/UnitsNet/wiki/String-Formatting